cli_explore.go

```diff
@@ -0,0 +1,48 @@
+package bc
```

Comment:
Not sure about this file and commands. Looks like its purpose is just to debug/test things.

Reply:
yep, this is some cli commands which helps me to understand the blockchain:-)